aws/config.go

```diff
@@ -8,14 +8,22 @@ type Config struct {
 	Region                string `hclext:"region,optional"`
 	Profile               string `hclext:"profile,optional"`
 	SharedCredentialsFile string `hclext:"shared_credentials_file,optional"`
+	AssumeRoleARN         string `hclext:"assume_role_arn,optional"`
+	AssumeRoleExternalID  string `hclext:"assume_role_external_id,optional"`
+	AssumeRolePolicy      string `hclext:"assume_role_policy,optional"`
+	AssumeRoleSessionName string `hclext:"assume_role_session_name,optional"`
```

Comment:
I think it's easier to understand if you can declare it in the `assume_role` block like Terraform. e.g:

```hcl
plugin "aws" {
  enabled = true

  assume_role {
    role_arn     = "..."
    external_id  = "..."
    policy       = "..."
    session_name = "..."
  }
}
```

Nested structures can be decoded as follows:
https://pkg.go.dev/github.com/terraform-linters/tflint-plugin-sdk@v0.14.0/hclext#example-DecodeBody

Reply:
Thank you for the information.
Added fix to declare in assume_role block like Terraform.